drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -245,31 +265,41 @@ public void onRedo() {
     }
 
     public void setUndoButtonEnabledStatus(boolean enabled) {
-        view.getUndoMenuItem().setEnabled(enabled);
+        undoMenuItem.setEnabled(enabled);
     }
 
     public void setRedoButtonEnabledStatus(boolean enabled) {
-        view.getRedoMenuItem().setEnabled(enabled);
+        redoMenuItem.setEnabled(enabled);
+    }
+
+    public void setItemMenuForMainGridEnabled(boolean enabled) {
+        runScenarioMenuItem.setEnabled(enabled);
+        importMenuItem.setEnabled(enabled);
+        exportToCSVMenuItem.setEnabled(enabled);
+        if (downloadMenuItem != null) {
+            downloadMenuItem.setEnabled(enabled);
+        }
```

Comment:
@danielezonca We agreed with @gitgabrio to **temporary** disable these commands when background is focused. In a future PR this will change. 
`downloadMenuItem` has the null check because is the only one initialized after a callback, all the others are initialized during presenter initialization.